Resources/config/services.xml

```diff
@@ -29,6 +29,24 @@
             <argument type="service" id="goaop.aspect.kernel"/>
             <argument type="service" id="goaop.cache.path.manager"/>
         </service>
+
+        <service id="goaop.command.warmup" public="false" class="Go\Symfony\GoAopBundle\Command\CacheWarmupCommand">
```

Comment:
In my company we are trying to extract service definitions into meaningful separate XML files. Here we can use `commands.xml` file for example and put all command definitions into this file. Just to keep it on the right place )